content/docs/dvclive/index.md

```diff
@@ -0,0 +1,19 @@
+# dvclive
+
+[dvclive](https://cml.dev) is an open-source python library for monitoring
+[dvclive](https://cml.dev) is an open-source python library for monitoring the
+progress of metrics during training of machine learning projects.
+
+dvclive is integrated seamlesly with dvc and logs produced by it can be fed to
+`dvc plots` command. Even though, one does not need dvc to visualize dvclive
+logs, as they are saved into easily parsable tsv format, feel free to apply
+custom visualization methods.
```

Comment:
```suggestion
Dvclive integrates seamlessly with DVC and the logs it produces can be fed as
`dvc plots`. However, `dvc` is not needed to work with `dvclive` logs, and since
they're saved as easily parsable TSV files, you can use your preferred
visualization method.
```